express.js

```diff
@@ -227,7 +228,7 @@ app.get('/login', passport.authenticate('faforever'));
 passport.use('faforever', new OidcStrategy({
     issuer: process.env.OAUTH_URL + '/',
     tokenURL: process.env.OAUTH_URL + '/oauth2/token',
-    authorizationURL: process.env.OAUTH_URL + '/oauth2/auth',
+    authorizationURL: process.env.OAUTH_PUBLIC_URL + '/oauth2/auth',
```

Comment:
a browser can not resolve the docker name aka "http://faf-ory-hydra:4444", so this is needed for docker environments, this needs also be fixed in the faf-stack